### ISSUE TITLE
Stream crossing over another stream 1.10.2

### ISSUE THREAD

**Issue by sharlin073** [delvr/Streams#44]
http://postfiles11.naver.net/MjAxNzA4MjdfMTc5/MDAxNTAzODE0OTU5MjQ3.zpY6OFKFfDwYIPAK-W4YszJBvEVsSpise3Q0JQfENTYg.QF805bUMaOzoqYURyBKtQL2J7SvUqqv5iODLiF48D5Ag.PNG.bluebird9281/2017-08-27_15.16.12.png?type=w2

If the image file is not opening I'll upload again sorry I'm new to this environment

Just saw a Stream crossing another stream in 1.10.2 with Realistic Terrain Generation mod.  I have WTF's expedition mod too. The game crashed just after taking that screenshot.

**Comment by delvr** [comment]
Yeah I've never been able to completely get rid of these crossing streams. :( They'll be gone after the Streams 1.0 rewrite but probably not before then. They shouldn't crash the game though - do you still have a log of the crash by any chance?

**Comment by sharlin073** [comment]
Sorry the crash was caused by WTF's expedition. thanks for replying! I'll be waiting for 1.0 :)
